Gemfile

```diff
@@ -182,13 +183,12 @@ gem 'dynamic_form'
 gem 'record_tag_helper', '~> 1.0'
 
 group :test do
-  # To remove once migrated all functional tests
-  gem 'codecov', :require => false
   gem 'rack-no_animations', '~> 1.0.3'
   gem 'rails-controller-testing', '~> 1.0.4'
-  gem 'simplecov', '~> 0.21.2', require: false
+  gem 'simplecov', '~> 0.22.0', require: false
```

Comment:
This change was because of this error seen in CircleCI:

```
rake aborted!
coverage measurement is already setup
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/simplecov-0.21.2/lib/simplecov.rb:354:in `start'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/simplecov-0.21.2/lib/simplecov.rb:354:in `start_coverage_with_criteria'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/simplecov-0.21.2/lib/simplecov.rb:343:in `start_coverage_measurement'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/simplecov-0.21.2/lib/simplecov.rb:59:in `start'
/opt/ci/workdir/test/minitest_helper.rb:18:in `<top (re
```

referenced in https://github.com/simplecov-ruby/simplecov/issues/1020 and fixed in https://github.com/simplecov-ruby/simplecov/pull/1035

This is included in `0.22.0` as per https://github.com/simplecov-ruby/simplecov/pull/1041#issuecomment-1364330557